web/src/test/manual/web/regression-tests/test.js

```diff
@@ -146,7 +141,6 @@ cleanKeyboards.then(() => {
           reject();
         }
         fs.copyFileSync(path.join(config.KEYMAN_REPO_BASE_RELATIVE_PATH, 'developer', 'bin', 'kmcomp.exe'), kmcompDestPath);
```

Comment:
Now I'm confused. I thought this PR would remove `kmcomp` ? :confused:

Reply:
You are right. I think this file needs more work -- there's a bunch of stuff that no longer makes sense here, and I missed it.

Reply:
I'm going to do this as a follow-up PR. It needs some significant review because it's working across multiple versions, but it probably also hasn't been used for nearly 5 years because it's a manual test, so it's not urgent.

Reply:
See #9651.